frontend/test/metabase/scenarios/sharing/subscriptions.cy.spec.js

```diff
@@ -26,7 +26,18 @@ describe("scenarios > dashboard > subscriptions", () => {
       .should("have.class", "cursor-default");
   });
 
-  it.skip("should allow sharing if dashboard contains only text cards (metabase#15077)", () => {
+  it("should not allow sharing if there are no dashboard cards", () => {
```

Comment:
no-op comment -- since this bug relies on a single array prop and still requires us to look at the implementation of the component I think this could probably become a really simple unit test at some point